source/IAccessibleHandler/__init__.py

```diff
@@ -602,20 +602,15 @@ def winEventToNVDAEvent(  # noqa: C901
 	return (NVDAEventName, obj)
 
 
-def processGenericWinEvent(eventID, window, objectID, childID):
+def processGenericWinEvent(eventID: int, window: int, objectID: int, childID: int):
```

Comment:
Please add return type hint
```suggestion
def processGenericWinEvent(eventID: int, window: int, objectID: int, childID: int) -> bool:
```